chapter_appendix-mathematics-for-deep-learning/distributions.md

```diff
@@ -82,7 +82,9 @@ We can sample an array of arbitrary shape from a Bernoulli random variable as fo
 
 ```{.python .input}
 #@tab pytorch
-1*(torch.randn(10, 10) < p)
+# Bernoulli distribution
+bernoulli = torch.distributions.bernoulli.Bernoulli(p)
+bernoulli.sample(sample_shape=(10,10))
```

Comment:
Just to make the codes consistent. We can simply use torch.rand here.
No need to use bernoulli from torch here.

Reply:
Let me change it back